packages/components/heading/.prettierignore

```diff
@@ -0,0 +1,12 @@
+build/
```

Comment:
Can we only keep the prettier configs in the root rather than having those files all over the components?